prettypandas/sparklines.py

```diff
@@ -0,0 +1,144 @@
+from jinja2 import Template
+import numpy as np
+
+
+class Sparkline(object):
+    """SVG Sparkline from numpy array.
+
+    :param data:
+        Array data to be used for line
+    :param width: default 150px
+        width of svg
+    :param height: default 20px
+        height of svg
+    :param height_offset:
+        offset from top/bottom to line
+    :param width_offset:
+        offset from sides to lines
+    :param show_max:
+        show green dot indicating maximums
+    :param show_min:
+        show red dot indicating minimums
+    """
+
+    TEMPLATE = """
+        {% macro circle(x, y, fill, class="", r=2) -%}
+            <circle cx="{{ x }}"
+                    cy="{{ y }}"
+                    r="{{ r }}"
+                    fill="{{ fill }}"
+                    class="{{ class }}"
+                    />
+        {%- endmacro %}
+
+        <svg width="{{width}}"
+             height="{{height}}"
+             version="1.1"
+             xmlns="http://www.w3.org/2000/svg">
+
+          <polyline points="
+            {%- for x, y in points -%}
+                {{ x }} {{ y }},
+            {%- endfor -%}"
+
+          class="line"
+          fill="transparent"
+          stroke="black" />
+
+          {% if not show_max and not show_min %}
+            {{ circle(points[-1][0], points[-1][1], "black", class="end") }}
+          {% endif %}
+
+          {% for x, y in maxs %}
+            {{ circle(x, y, "green", class="max", r=2)}}
+          {% endfor %}
+
+          {% for x, y in mins -%}
+            {{ circle(x, y, "red", class="min", r=2)}}
+          {%- endfor %}
+
+        </svg>
+        """
```

Comment:
The Jinja2 style was inspired by the Pandas `Styler` class using templates. I could have used one of the python SVG libraries but decided not to bother with another API and dependency for such a simple use case.